src/common/hooks/use-selected-asset.ts

```diff
@@ -30,11 +32,11 @@ export function useSelectedAsset() {
   const balance = useMemo<string | undefined>(() => {
     if (!selectedAsset) return;
     if (selectedAsset.type === 'stx')
-      return stacksValue({ value: selectedAsset.balance, withTicker: false });
+      return stacksValue({ value: availableStxBalance?.toString() || 0, withTicker: false });
```

Comment:
```suggestion
      return stacksValue({ value: availableStxBalance || 0, withTicker: false });
```

Reply:
Ok, I see this now with the changes Kyran made adding the `BigNumber` type to value. I'm just getting familiar with using BigNumber. Will go thru and makes these changes.